### PR TITLE
ElasticsearchOSP tests

### PR DESCRIPTION
**Pull request by adrianfusco**
- Add list of allowed queries to ElasticSearchOSP.
- Add test for `ElasticSearchOSP.get_jobs_by_name` method.

### CODE REVIEW

File: tests/sources/elasticsearch/test_api.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+from unittest.mock import Mock
+
+from cibyl.sources.elasticsearch.api import ElasticSearchOSP
+
+
+def return_hits():
+    """It will return fake hits from elasticsearch
+    to use it in a Mock
+    """
+    return [
```
Comment (bregman-arie) [inline tests/sources/elasticsearch/test_api.py:26]: seems to be something you can include in def setUp as:

self.return_hits = {
            '_index': 'test',
            '_id': 'random',
            '_score': 1.0
        }

wdyt?

Comment (adrianfusco) [reply tests/sources/elasticsearch/test_api.py]: You're right. I did the change.

**Comment by adrianfusco** [comment]
I've finished the changes. It's done for review.

**Review by jsanemet — APPROVED** [review]

**Review by cescgina — APPROVED** [review]
Looks good, maybe I would add a test mocking es_client.search to actually test __query_get_hits

**Comment by bregman-arie** [comment]
Looks good. You got the approvals to merge, so feel free to either make the change Joan suggested or merge it as it is and do it in a future PR.

**Comment by adrianfusco** [comment]
> Looks good. You got the approvals to merge, so feel free to either make the change Joan suggested or merge it as it is and do it in a future PR.

You're right @cescgina.

I'll merge it @bregman-arie. I should add more changes for get tests from elasticsearch so I'll add more tests.

Thx!

**Pull request merged**